### DISCUSSION TITLE
Upgrading to latest Cloduy

### DISCUSSION THREAD

**Discussion by Cippo2019** [krahabb/meross_lan#294] [General]
Hi Krahabb , thank in advance for your great job
I have read into info that shoudl be "transparent" the update of tha Meross_lan from 3.x to 4.x 
Only if you configure clound account you can start to use new feature, right ?
I am asking to be sure, if i cloud :-) , before upgrade because now all my meross devices are configured on normal (or we can say old) way and I want to keep as it is and only add a new account for some other new meross device on another house.

thanks in advance
Tiziano

**Comment by krahabb** [comment]
Yep! Thank you @Cippo2019,
By not configuring any cloud profile the integration works as before, it might bother you by prompting the discovery af an 'unknown cloud profile' but you can just 'ignore' it

**Comment by Cippo2019** [comment]
Done !
it was like you said :-)
thanks a lot again for that super work

wbr
Tiziano
